.eslintignore

```diff
@@ -0,0 +1,2 @@
+/modules/books.js
+/modules/bookActions.js
```

Comment:
- [ ] As I've requested before, kindly ensure that `eslint checks` are always running in the modules that you create. There's always a way to make the Linters work and they'll seem hard the first time, but once you get in the habit of following the practices you'll see that they're indeed helpful and will make you write much better code. [Here is a resource](https://sourcelevel.io/blog/what-is-a-linter-and-why-your-team-should-use-it) that shows the many advantages of using linters. Please remove these modules in the `.eslintignore` file and fix the linter errors in your modules.